src/org/smssecure/smssecure/ConversationActivity.java

```diff
@@ -970,6 +1022,7 @@ private Drafts getDraftsForCurrentState() {
     for (Slide slide : attachmentManager.getSlideDeck().getSlides()) {
       if      (slide.hasAudio()) drafts.add(new Draft(Draft.AUDIO, slide.getUri().toString()));
       else if (slide.hasVideo()) drafts.add(new Draft(Draft.VIDEO, slide.getUri().toString()));
+      else if (slide.hasFile()) drafts.add(new Draft(Draft.FILE, slide.getUri().toString()));
```

Comment:
This line of code can use some comment; because the order here is pretty important. hasImage() needs to be the last one, as it is also used to determine the fact if a thumbnail needs to be shown (hasImage is true on audio and file).